packages/components/src/components/text-field-wrapper/__snapshots__/text-field-wrapper-styles.spec.ts.snap

```diff
@@ -46,23 +45,22 @@ exports[`getComponentCss() should return correct css for isDisabled: false, hide
   -webkit-background-clip: padding-box !important;
 }
 @media(hover:hover) {
-  ::slotted(input:not(:disabled):not(:focus):not([readonly]):hover) {
+  ::slotted(input:not(:disabled):not(:focus):not([readonly]):hover),label:hover~* ::slotted(input:not(:disabled):not(:focus):not([readonly])),label:hover~* ::part(p-select-wrapper-dropdown) {
```

Comment:
Unrelated, new styles?

Reply:
It's not unrelated. In my opinion correct here, otherwise the style would be duplicated if you want to separate the selectors.

Reply:
select-wrapper hover selector is conditional now.